src/assets/translations/en/main.json

```
@@ -600,8 +601,20 @@
         "exit": "Unstake",
         "removeLiquidityETH": "Remove liquidity"
       },
-      "thor": {
-        "transferOut": "Transfer out"
+      "thorchain": {
+        "deposit": "Deposit",
+        "depositRefund": "Deposit Refund",
+        "loanOpen": "Open Loan",
```

If we want to stick to the same verbage as e.g "Deposit Refund" above

```suggestion
        "loanOpen": "Loan Open",
```

---

https://github.com/shapeshift/web/pull/5986/commits/f58e07a45534dc01157b56082a72de7b4d8ded5f